models/aavegotchi/polygon/aavegotchi_polygon_fees.sql

```diff
@@ -0,0 +1,43 @@
+ {{
+  config(
+        alias='fees',
+        post_hook='{{ expose_spells(\'["polygon"]\',
+                                    "project",
+                                    "aavegotchi",
+                                    \'["springzh"]\') }}')
+}}
+
+SELECT blockchain,
+project,
+version,
+block_time,
+token_id,
+collection,
+CAST(platform_fee_amount_raw AS DOUBLE) AS platform_fee_amount_raw,
+platform_fee_amount,
+platform_fee_amount_usd,
+CAST(platform_fee_percentage AS DOUBLE) AS platform_fee_percentage,
+CAST(royalty_fee_amount_raw AS DOUBLE) AS royalty_fee_amount_raw,
+royalty_fee_amount,
+royalty_fee_amount_usd,
+CAST(royalty_fee_percentage AS DOUBLE) AS royalty_fee_percentage,
+royalty_fee_receive_address,
+royalty_fee_currency_symbol,
+token_standard,
+trade_type,
+CAST(number_of_items AS DECIMAL(38,0)) AS number_of_items,
```

Comment:
i notice this is casted as int in `events` then decimal here (which is correct type). please move this to `events` and override the int casting

Reply:
Updated.